telemetry/opentelemetry-config/src/main/java/module-info.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.helidon.common.features.api.Features;
+import io.helidon.common.features.api.HelidonFlavor;
+
+/**
+ * OpenTelemetry implementation for telemetry.
+ */
+@Features.Name("Telemetry")
+@Features.Description("Support for Telemetry")
+@Features.Flavor({HelidonFlavor.SE, HelidonFlavor.MP})
+@Features.Path({"Telemetry"})
+@Features.Incubating
+module io.helidon.telemetry.otelconfig {
+
+    requires io.helidon.builder.api;
+    requires io.helidon.common.configurable;
+    requires io.helidon.config;
+    requires io.helidon.service.registry;
+
+    requires io.opentelemetry.api;
```

Comment:
There are a few dependencies declared as `optional`, yet here they are as `requires` - all optional dependencies must be declared as `requires static`, as otherwise they may not be available at runtime, and this module would not load.

Reply:
Fixed.